app/controllers/books_controller.rb

```diff
@@ -1,11 +1,5 @@
 class BooksController < ApplicationController
```

Comment:
Looks like this controller can be deleted now

Reply:
No, we need it. What we will have root without it?

Reply:
routes.rb file

Reply:
Sorry I mean why do you need this controller? Do you render vue component in `BooksController#index`?

Reply:
I suppose, that @isabloCh gonna create individuals Vue component for every page

@Loremaster, Do you mean a single component to rule them all?

Reply:
yes

Reply:
@resivalex yep I think it’ll be good to have it. We don’t have to do it in this PR